src/analytics/block_activity.rs

```diff
@@ -0,0 +1,44 @@
+// Copyright 2023 IOTA Stiftung
+// SPDX-License-Identifier: Apache-2.0
+
+use super::BlockAnalytics;
+use crate::types::{
+    ledger::BlockMetadata,
+    stardust::block::{Block, Output, Payload},
+    tangle::MilestoneIndex,
+};
+
+#[derive(Clone, Debug, Default)]
```

Comment:
should be fine to make this type also `Copy`

Reply:
I think for now I like the explicitness of the copy until we have settled on a good API for returning the measurements. Performance-wise it shouldn't make a difference.